### PR TITLE
rectangle in the circle

### PR DESCRIPTION
**Pull request by mujeeb100**
plesase review it small css task.

### CODE REVIEW

**Review by shaikh-7abish — APPROVED** [review]

**Review by AlishaPrasad — APPROVED** [review]

**Review by faran4engg — CHANGES_REQUESTED** [review]
correct folder name 😒

File: tasks-by-users/muzeeb100/rectangle in the circle/style.css [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  position: absolute;
+  content: "";
+  top: 50%;
+  left: 50%;
+  width: 300px;
+  height: 300px;
+  background-color: red;
+  border-radius: 50%;
+  transform: translate(-50%, -50%);
+  border: 5px solid black;
```
Comment (faran4engg) [inline tasks-by-users/muzeeb100/rectangle in the circle/style.css:25]: group properties

File: tasks-by-users/muzeeb100/rectangle in the circle/style.css [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+}
+
+.circle {
+  border: 5px solid black;
+  left: 25%;
+  top: 100px;
+  position: relative;
+  width: 50%;
+  height: 500px;
+  background-color: orange;
```
Comment (faran4engg) [inline tasks-by-users/muzeeb100/rectangle in the circle/style.css:13]: here too

**Comment by faran4engg** [comment]
> lgtm

How come ? @Tauqeer-Ahmed-99 !
you overlooked css grouping + file name in the PR (name has spaces)

**Review by nehalskdev — APPROVED** [review]

**Pull request merged**